Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -697,6 +697,31 @@ theorem integrable_neg_iff {f : α → β} : Integrable (-f) μ ↔ Integrable f
   ⟨fun h => neg_neg f ▸ h.neg, Integrable.neg⟩
 #align measure_theory.integrable_neg_iff MeasureTheory.integrable_neg_iff
 
+@[simp]
+lemma integrable_add_const_iff [IsFiniteMeasure μ] {f : α → β} {c : β} :
+    Integrable (fun x ↦ f x + c) μ ↔ Integrable f μ :=
+  ⟨fun h ↦ show f = fun x ↦ f x + c + (-c) by simp only [add_neg_cancel_right]
+    ▸ h.add (integrable_const _), fun h ↦ h.add (integrable_const _)⟩
+
+@[simp]
+lemma integrable_const_add_iff [IsFiniteMeasure μ] {f : α → β} {c : β} :
+    Integrable (fun x ↦ c + f x) μ ↔ Integrable f μ :=
+  ⟨fun h ↦ show f = fun x ↦ (c + f x) + (-c)
+    by simp only [add_neg_cancel_comm] ▸ Integrable.add h (integrable_const (-c)),
+    fun h ↦ Integrable.add (integrable_const _) h⟩
```

Comment:
does the tactic `simp_rw [add_comm, integrable_add_const_iff]` do this?

Reply:
'simp_rw' does not manage to apply the 'add_comm' in this situation, so I used 'integrable_add_iff_integrable_left', symmetrically to the 'integrable_const_add_iff' case

Reply:
It will probably work if you write `add_comm c`. But your current proof is also short, so no need to change.

bors merge